src/core/read_halos-velociraptor.c

```diff
@@ -339,7 +349,25 @@ void read_trees__velociraptor(int snapshot,
               fof_group->Mvir = tree_entry.Mass_200crit;
               fof_group->Rvir = tree_entry.R_200crit;
             }
+          }*/
+          
+          // Part below still work in progress, need to make sure of few things
+          
```

Comment:
is this part duplicated?

Reply:
Ups! Yes I didn't notice that, I am removing it!